source/delayed_capture/index.html.md.erb

```diff
@@ -18,27 +18,34 @@ You cannot use delayed capture for:
 * payments made with [payment links](https://www.payments.service.gov.uk/govuk-payment-pages/)
 * [recurring payments](/recurring_payments)
 
-To use this feature, include `"delayed_capture": true` in the body of a [Create new payment](/making_payments/#creating-a-payment) request.
+To use this feature, include `"delayed_capture": true` in the body of your [request to create a new payment](/making_payments/#creating-a-payment):
+
+`POST /v1/payments`
```

Comment:
Is this code an example of using "delayed_capturew": true? It may just be me, but I wonder if you need to make that clear. To me at the moment it feels like you're saying "include this" then posting some code a bit out of context.

Reply:
Perfect! And yes, your other example lead-ins are great. That's why this stood out a bit.